includes/commerce_pos.payment.inc

```diff
@@ -368,7 +368,7 @@ function commerce_pos_payment(&$form, &$form_state, $transaction_type) {
 
   $form['parked_transactions'] = commerce_pos_parked_transactions_section($transaction_type, $form_ajax, !empty($form_state['transaction']));
 
-  $form['#attached']['js'][] = drupal_get_path('module', 'commerce_pos') . '/js/commerce_pos.payment.js';
+  $form['#attached']['js'][] = drupal_get_path('module', 'commerce_pos') . '/js/commerce_pos.js';
```

Comment:
Explanation in [this discussion](https://github.com/AcroMedia/commerce_pos/pull/28#discussion_r109998696).